Controls/ConnectionControl.cs

```diff
@@ -164,7 +164,7 @@ private void cmb_sysid_Format(object sender, ListControlConvertEventArgs e)
                             mavComponentString = mavComponentString.Remove(0, mavComponentHeader.Length);
                         }
                     }
-                    e.Value = ((int)temp.sysid) + "-" + mavComponentString.Replace("_"," ");
+                    e.Value = ((int)temp.sysid) + "-" + mavComponentString.Replace("_"," ") + " on " + temp.port.BaseStream.PortName;

```

Comment:
any reason for this ordering?
ie when trying to find something having the broadest to narrowest makes sense. ie a drill down approach

this is currently adhoc

broadest being port then sysid then component

Reply:
This depends on which use-case you believe is more common: selecting between multiple systems (or components) communicating on a single port, or selecting between multiple ports connected to a single system.

In my opinion, selecting between ports shouldn't even be exposed to the user in this manner: if an aircraft has multiple communications links, they should be aggregated together, rather than treating them separately (refer #1481 ).